packages/remix-dev/config.ts

```diff
@@ -336,6 +337,7 @@ export async function readConfig(
     case "cloudflare-pages":
     case "cloudflare-workers":
     case "deno":
+    case "fastly-compute-js":
```

Comment:
Instead of adding this line, use `serverModuleFormat` and `serverPlatform` directly in your remix.config.js in your template